test/libraries/Pool.t.sol

```diff
@@ -76,14 +81,17 @@ contract PoolTest is Test {
         state.modifyLiquidity(params);
     }
 
-    function testSwap(uint160 sqrtPriceX96, uint24 swapFee, Pool.SwapParams memory params) public {
+    function testSwap(uint160 sqrtPriceX96, uint24 swapFee, uint24 protocolFee, Pool.SwapParams memory params) public {
         // Assumptions tested in PoolManager.t.sol
         params.tickSpacing = int24(bound(params.tickSpacing, TickMath.MIN_TICK_SPACING, TickMath.MAX_TICK_SPACING));
```

Comment:
Since tickSpacing is always going to be positive (since MIN_TICK_SPACING is 1), should we make tickSpacing in SwapParams a uint24 instead of an int24?